dask_mpi/core.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+from mpi4py import MPI
+from tornado.ioloop import IOLoop
+
+from dask_mpi.common import get_host_from_interface, start_scheduler, start_worker
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+loop = IOLoop()
+
+
+def initialize(scheduler_file='scheduler.json', interface=None, nthreads=1, local_directory='',
+               memory_limit='auto', scheduler=True, nanny=False, bokeh_port=8787, bokeh_prefix=None,
+               bokeh_worker_port=8789):
```

Comment:
Do we need the `scheduler=` keyword?  I suspect that this may just be a holdover from the last implementation.

Reply:
We probably do not.  I can remove it.

For similar reasons, do you think that `nanny=` and the `bokeh_...=` keywords should be removed, too?

Reply:
I think that nanny and bokeh will still be useful.

Reply:
Fair enough.  Can bokeh be disabled?  My understanding is that it is enabled by default, but I don't know how to disable it without uninstalling bokeh.

Reply:
I would hope that passing `bokeh=False` here would disable bokeh.

Reply:
It is run by default though if it is installed.  It's decently cheap to keep running in the background.

Reply:
For the bokeh part, I think that it is nice to be able to connect to Dask Dashboard on long running batches. Sometimes I want to check that everything goes fine even if I'm not doing interactive stuff.

So 👍 on keeping the availability to have bokeh, and defaulting with it sounds good.

Reply:
Optionnaly, we may want to use `initialize()` in cli.py too. `dask-mpi` would just be a particular case of user script? This could make things easier if we want to implement a `--script` option... 

But we can keep that for later.

Reply:
I've removed the `scheduler=` parameter from the `initialize` function (https://github.com/dask/dask-mpi/pull/6/commits/548556dc2836c7cecb04e4ba6cf11726773a8758)

Reply:
I don't think that using `initialize` in the CLI adds any value.  All of the common code (common between the CLI (in `cli.py`) and the `initialize` function (in `core.py`) can be found in the `common.py` module.  There is no code duplication.

Reply:
Is this issue resolved?

Reply:
I added a `bokeh=` parameter, per @mrocklin's suggestion.  I think that covers this conversation.